.github/workflows/ci.yml

```diff
@@ -34,14 +34,13 @@ jobs:
       - name: Build Framework
         run: swift package generate-xcodeproj --output generated/ && carthage build --project-directory generated/ --verbose --no-skip-current
   spm-11:
-    name: SPM Build Xcode 11
-    runs-on: macOS-10.15
+    name: Build Xcode 11
+    runs-on: macOS-11
     strategy:
       matrix:
         platforms: [
-          'iOS_12,tvOS_12',
```

Comment:
This sucks, but [Github Actions' macOS 11 install does not have iOS or tvOS 12 simulators](https://github.com/actions/runner-images/blob/main/images/macos/macos-11-Readme.md)